components/service/glean/src/test/java/mozilla/components/service/glean/GleanTest.kt

```diff
@@ -288,4 +289,32 @@ class GleanTest {
         gleanSpy.initialized = false
         gleanSpy.handleEvent(Glean.PingEvent.Default)
     }
+
+    @Test
```

Comment:
We should also add a test for making sure pings are not sent if there's no content.